public/app/plugins/datasource/prometheus/components/PromQueryEditorByApp.test.tsx

```diff
@@ -96,24 +96,4 @@ describe('PromQueryEditorByApp', () => {
     expect(getByTestId('QueryEditorModeToggle')).toBeInTheDocument();
     expect(queryByTestId(alertingTestIds.editor)).toBeNull();
   });
-
```

Comment:
I moved these tests to public/app/plugins/datasource/prometheus/components/PromQueryField.test.tsx si it is not dependant of initial query builder mode